plugins/state_history_plugin/state_history_plugin_abi.cpp

```diff
@@ -330,7 +330,7 @@ extern const char* const state_history_plugin_abi = R"({
             ]
         },
         {
-            "name": "chain_config_v0", "fields": [
+            "name": "chain_config", "fields": [
                 { "type": "uint64", "name": "max_block_net_usage" },
```

Comment:
Why was this change made?

Reply:
This needs to be changed back, all actual types have version numbers in the type definition

Reply:
Done.